src/main/scala/com/google/cloud/spark/bigquery/direct/DirectBigQueryRelation.scala

```diff
@@ -53,6 +53,13 @@ private[bigquery] class DirectBigQueryRelation(
   }
   private val log = Logger(getClass)
 
+  /**
+   * Default parallelism to 1 reader per 400MB, which should be about the maximum allowed by the
+   * BigQuery Storage API. The number of partitions returned may be significantly less depending
+   * on a number of factors.
+   */
+  val DEFAULT_BYTES_PER_PARTITION = 400L * 1000 * 1000
```

Comment:
How about making this user configurable? That way, you can provide users a better way to control parallelism for the case when they are selecting only a few columns.